examples/ads.amp.html

```diff
@@ -190,12 +191,20 @@ <h2 id="aduptech">Ad Up Technology</h2>
       data-adtest="1">
   </amp-ad>
 
-  <h2 id="amoad" class="broken">AMoAd</h2>
+  <h2 id="amoad-banner" class="broken">AMoAd banner</h2>
   <amp-ad width="300" height="250"
       type="amoad"
+      data-ad-type="banner"
       data-sid="62056d310111552c951d19c06bfa71e16e615e39493eceff667912488bc576a6">
   </amp-ad>
 
+  <h2 id="amoad-native" class="broken">AMoAd native</h2>
```

Comment:
We basically allow one h2 tag per ad network. This is for quick navigation only (see links at the top).
Please remove this tag.